upbound_uxp/manifest.json

```diff
@@ -0,0 +1,71 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f1427ca4-4b88-484a-a7c5-9cc66542b0d9",
+  "app_id": "upbound-uxp",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect and graph Upbound UXP metrics",
+    "title": "Upbound UXP",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Upbound",
+        "image_url": "images/upbound_uxp_logo.svg"
```

Comment:
Only png or jpg files are allowed. Please include a screenshot of your dashboard populated with data as well as any relevant screenshots of the Upbound platform. The logo will automatically be added to your tile and can be removed from the manifest.json file. 

Please see our [media guidelines here](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel) for requirements.

Reply:
Using png.